### ISSUE TITLE
public IPv6 addresses

### ISSUE THREAD

**Issue by Manawyrm** [freifunkh/ansible-configs#35]
Further discussion about rDNS, the use of the ULA and maybe even dynamic/all-knowing-DNS is still needed.

See: https://apps.db.ripe.net/db-web-ui/#/query?bflag&searchtext=f.f.0.0.0.9.7.0.2.0.a.2.ip6.arpa&source=RIPE&types=rtr-set;domain#resultsSection

Currently 
nserver:         pns01.dcsix.net
nserver:         pns02.dcsix.net
nserver:         pns03.dcsix.net
nserver:         pns04.dcsix.net
are responsible for delivering a reverse-dns lookup for the IPv6-address block.
In theory, we could (and maybe should) create a NS record on these nameservers, which only redirect the  2a02:790:ff:7/64 to our nameservers, aka ns1.fnorden.net and slaves. 
We could then go from there and (for example) set up an all-knowing-dns.

**Comment by lemoer** [comment]
hm, I see troubles when we support multidomain and when we use multiple supernodes to announce different subnets. This will result in a `/64` per supernode and per domain.

**Comment by Manawyrm** [comment]
I think that's fine. As long as all the different /64 subnets are in a common larger subnet, we can easily delegate the larger /56 or something, too. 
Only the all-knowing-dns would then need to know which net is which, but that isn't too big of a deal, too.

**Comment by lemoer** [comment]
Yes, the /64 are all part of a the larger subnet 2a02:790:ff::/48 .
